src/features/unified-share-modal/ContactRestrictionNotice.scss

```diff
@@ -0,0 +1,24 @@
+@import '../../styles/variables';
+
+.bdl-ContactRestrictionNotice {
+    .bdl-SelectButton {
+        margin-top: 10px;
```

Comment:
Oh ok, wasn't aware of that variable, let me update

Reply:
Updated most margins to use `bdl-grid-unit`, but the one for the loading indicator needs to be exact in order to prevent a jump when loading completes.